lineapy/plugins/pipeline_writers.py

```diff
@@ -92,13 +92,14 @@ def _write_requirements(self) -> None:
             for lib in session_libs:
                 libraries[lib.package_name] = lib.version
         lib_names_text = "\n".join(
-            [f"{lib}=={ver}" for lib, ver in libraries.items()]
+            [
+                lib if lib == "lineapy" else f"{lib}=={ver}"
+                for lib, ver in libraries.items()
+            ]
```

Comment:
@lionsardesai: This change is to retain what exists in our current codebase where `lineapy`'s version is not specified in `requirements.txt`. If I remember correctly, this had been done to avoid version conflict in testing. But I now wonder if this is something we want for the user, who may actually need the specific `lineapy` version listed in their `requirements.txt`; otherwise, the latest version of the package will be installed and this may cause compatibility issues.

Reply:
lets tackle that outside this PR.